.code-samples.meilisearch.yaml

```diff
@@ -91,6 +92,10 @@ update_settings_1: |-
       "genre",
       "release_date",
     },
+    SortableAttributes: []string{
+      "title",
+      "release_date",
+    },
```

Comment:
Just a detail, but the other code sample are after `stop-words`, before `synonyms` -> https://github.com/meilisearch/documentation/blob/1aac9329942346353316108cfa53adcd0abc3993/.code-samples.meilisearch.yaml#L118-L121